modules/hosted-restart-hcp-components.adoc

```diff
@@ -10,21 +10,26 @@ If you are an administrator for hosted control planes, you can use the `hypershi
 
 .Procedure
 
-To restart a control plane, annotate the `HostedCluster` resource by entering the following command:
-
+* To restart a control plane, annotate the `HostedCluster` resource by entering the following command:
++
 [source,terminal]
 ----
 $ oc annotate hostedcluster \
   -n <hosted_cluster_namespace> \
   <hosted_cluster_name> \
-  hypershift.openshift.io/restart-date=$(date --iso-8601=seconds)
+  hypershift.openshift.io/restart-date=$(date --iso-8601=seconds) <1>
 ----
+<1> The `date` command serves as the source of a unique string. The annotation is treated as a string, not a timestamp.
+
 
 .Verification
 
-The control plane is restarted whenever the value of the anonotation changes. The `date` command in the example serves as the source of a unique string. The annotation is treated as a string, not a timestamp.
+The control plane is restarted whenever the value of the annotation changes.
 
-The following components are restarted:
+[NOTE]
+====
+After you restart a control plane, the following {hcp} components are typically restarted. However, you might see some additional components restarting as a side effect of changes implemented by the other components.
+====
```

Comment:
Suggestion to consider - it might be a little odd that the list below is now introduced by a sentence contained within a note. If users skip over the note, there is now just a list mentioned for reasons they don't understand. Could you maybe pull the first sentence out of the note?  You could definitely keep the second sentence in there imo.

Here's what that might look like:
```suggestion
After you restart a control plane, the following {hcp} components are typically restarted:
[NOTE]
====
You might see some additional components restarting as a side effect of changes implemented by the other components.
====
```

Reply:
@skopacz1 I like this idea. Thank you!